cmd/meroxa/root/apps/deploy.go

```diff
@@ -66,6 +66,22 @@ type deployApplicationClient interface {
 	AddHeader(key, value string)
 }
 
+type environment struct {
+	Name string
+	UUID string
+}
+
+func (e *environment) nameOrUUID() string {
+	switch {
+	case e.UUID != "":
+		return e.UUID
+	case e.Name != "":
+		return e.Name
+	default:
+		panic("bad state: name or uuid should be present")
```

Comment:
i prefer not to panic because it feels like a bad customer experience if the CLI were to ever actually hit this

Reply:
I thought about this for a bit, the panic is a result of very unexpected state and we should be very vocal about it. Indeed this should never happen, but if it does.. it will blow up spectacularly, panic is used seldom, but it is used in the code base.

Reply:
Re bad customer experience, I'm not too worried, since this will grab the users attention and there is a higher chance of reporting it and engaging in a discussion.

At least this is my reasoning, let me know if you think its unreasonable.

Reply:
Sorry, I missed your responses.  I don't think customers appreciate having to sort through stacktraces.  "Error: bad state" is as clear, imho.

Reply:
Yes, but it doesn't provide context why the state is bad. Ideally we want them to file a bug.